docs/logparse/parse-unknown-labels.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import json
+import re
+from argparse import Namespace
+
+import pandas as pd
+
+shop_regex = re.compile("(?<=INFO - extract.utils: unknown sustainability label from ).*(?=:)")
+
+
+def parse_args() -> Namespace:
+    """
+    Parses the cmd arguments.
+
+    Returns:
+        argparse.ArgumentParser: Object containing the parsed arguments.
+    """
+    argparser = argparse.ArgumentParser(description="process arguments")
+    argparser.add_argument("-f", "--file", type=str, help="Absolute path to logfile to parse.")
+    return argparser.parse_args()
+
+
+def main() -> None:
+    """
+    Main method to parse an extractor logfile for unknown sustainability labels.
+    """
+    args = parse_args()
+    infile = args.file
+    print(f"Parsing {infile}")
+
+    with open(infile, encoding="utf-8") as f:
+        file = f.readlines()
+
+    shop_labels = []
+
+    for line in file:
+        if shop := shop_regex.findall(line):
+            shop = shop[0]
+            unknown_label = re.findall(f"(?<={shop}: ).*", line)
+            shop_labels.append((shop, unknown_label[0]))
+
+    df = pd.DataFrame(shop_labels, columns=["shop", "label"])
+    df_stats = pd.DataFrame(df.value_counts()).sort_values(by=["shop", 0], ascending=[False, False])
```

Comment:
Is 0 the index here?

Reply:
No, it is the column created from `value_counts()`.

Reply:
Ah I see, then maybe it's helpful to leave a comment on top of that line to know what it's doing

Reply:
Yes, you are right. I optimized the code a little, as the value count was actually not necessary and added a comment.